### PR TITLE
[ENHANCEMENT] Let ember install take multiple addons

### PR DESCRIPTION
**Pull request by DanielOchoa**
As the title says, it lets `ember install` take multiple addon names. Fixes half of #4176

### CODE REVIEW

**Comment by DanielOchoa** [comment]
well, it seems that the windows env does not like one of the tests... I'll dig into this sometime today

File: lib/tasks/addon-install.js [hunk 1]
```diff
@@ -31,33 +32,37 @@ module.exports = Task.extend({
       testing:    this.testing
     });
 
-    ui.startProgress(chalk.green('Installing addon package'), chalk.green('.'));
+    var packageInflection = inflector.inflect('package', packageNames.length);
```
Comment (stefanpenner) [inline lib/tasks/addon-install.js]: do we need a full inflection library for this? Seems like this inflection is trivial no?

Comment (DanielOchoa) [inline lib/tasks/addon-install.js]: It is.. now that mention it.

**Comment by stefanpenner** [comment]
I really like this, once CI is green we should :shipit:

**Comment by DanielOchoa** [comment]
awesome.

**Comment by trabus** [comment]
I restarted the appveyor build, that timeout happens occasionally.

**Comment by DanielOchoa** [comment]
The timeout is related to my pr, more precisely the slow test to ember
install. Windows does not like (times out) when I try to rm the tmp dir.

**Comment by DanielOchoa** [comment]
So let me know if you have any ideas. :)

**Comment by stefanpenner** [comment]
If this persists it is likely a real bug. To test on windows https://github.com/ember-cli/ember-cli/tree/master/dev/windows may be of help

**Comment by stefanpenner** [comment]
@DanielOchoa any luck?

**Comment by DanielOchoa** [comment]
Not yet. The VM I setup from the link doesn't run any tests (it just hangs up). I'll spend some time this evening digging more through it.

**Comment by kellyselden** [comment]
@DanielOchoa I created that windows VM code, I might be able to help you with any issues.

**Comment by DanielOchoa** [comment]
That would be awesome.

File: tests/acceptance/install-test-slow.js [hunk 2]
```diff
@@ -7,32 +7,29 @@ var assertFile = require('../helpers/assert-file');
...
-  beforeEach(function() {
-    tmpdir = tmp.in(tmproot);
-    process.chdir(tmpdir);
+    process.chdir(root);
   });
 
   afterEach(function() {
-    process.chdir(root);
-    return remove(tmproot);
+    return remove(tmproot, { maxBusyTries: 10 });
```
Comment (DanielOchoa) [inline tests/acceptance/install-test-slow.js]: The test fails exactly on this line. (On appveyor.) It returns a busy signal error.

Comment (DanielOchoa) [inline tests/acceptance/install-test-slow.js]: Well actually, with the maxBusyTries prop set to 10 it times out, when the prop is not set (defaults to 3) it returns the busy signal error.

**Comment by DanielOchoa** [comment]
In other words, I'm very positive that the change works well on windows, it's just `tests/acceptance/install-test-slow.js` giving us a hard time in windows. As I mentioned before, I'll look at it again later today if @kellyselden hasn't found a fix. :)

**Comment by stefanpenner** [comment]
@DanielOchoa hmm, so we need to figure out how to get CI green :(

**Comment by DanielOchoa** [comment]
we'll get there. I'll take a break and look into it right now.

**Comment by stefanpenner** [comment]
maybe we are accidentally creating a much more massive install then expected. We do some funky business to minimize cost of `npm` installs (lots of symlinking) maybe we are tripping something up.

**Comment by DanielOchoa** [comment]
That seems related yes. The tmp dir in that specific test is created through regular means (I think) so I'll keep that in mind when debugging.

**Comment by stefanpenner** [comment]
its green!

**Comment by kellyselden** [comment]
:+1:

**Pull request merged**